ledger-service/http-json/src/main/scala/com/digitalasset/http/PackageService.scala

```diff
@@ -370,11 +375,17 @@ object PackageService {
   final case class ContractTypeIdMap[CtId[T] <: ContractTypeId[T]](
       all: Map[RequiredPkg[CtId], ResolvedOf[CtId]],
       unique: Map[NoPkg[CtId], ResolvedOf[CtId]],
+      nameIds: Map[Ref.PackageName, String], // TODO(ETX-451): Multiple package ids per package name
   ) {
+    private def packageRefToPackageId(pkgRef: String): Option[String] = pkgRef match {
```

Comment:
I would be tempted to avoid PackageRef -> Set[PackageId] mappings because if risk of an incomplete set of package-ids. The above is fine for this limited-support PR.

Reply:
That mapping needs to be somewhere. Are you saying you'd lean towards having it in the db over memory?